lib/provider.js

```diff
@@ -33,14 +33,18 @@ module.exports = {
 
   getSuggestions({editor, bufferPosition, scopeDescriptor}) {
     const prefix = getPrefix(editor, bufferPosition);
+    const filePath = editor.getPath();
 
     if (shouldPreventAutocomplete(prefix)) {
       return null;
     }
 
+    if (filePath === undefined) {
```

Comment:
Nice! This looks like the root cause of the problem. Let's go ahead and clean it up just a bit. Rather than having another `if` statement here that could return null, let's go ahead and pass the `filePath` into the `shouldPreventAutoComplete` function and do all of the checks in there.

e.g.

```js
if shouldPreventAutoComplete(prefix, filePath) {
  return null;
}
```

And then `shouldPreventAutoComplete` could looks like this:

```js
const shouldPreventAutocomplete = (prefix, filePath) => {
  return prefix.length < atom.config.get('language-elm.minCharsForAutocomplete')
      || !atom.config.get('language-elm.autocompleteEnabled')
      || !filePath; // This is the important part! No need to check explicitly for undefined, as undefined is a "falsy" value
}
```